openeo/Pre&PostCovid.ipynb

```diff
@@ -0,0 +1,358 @@
+    "        {\n",
+    "            \"type\": \"Feature\",\n",
+    "            \"properties\": {},\n",
+    "            \"geometry\": {\n",
+    "                \"coordinates\": [\n",
+    "                    [\n",
+    "                        [77.11, 28.69],\n",
+    "                        [77.11, 28.56],\n",
+    "                        [77.29, 28.56],\n",
+    "                        [77.29, 28.69],\n",
+    "                        [77.11, 28.69],\n",
+    "                    ]\n",
+    "                ],\n",
+    "                \"type\": \"Polygon\",\n",
+    "            },\n",
+    "        }\n",
+    "    ],\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "e504a43b-37c6-4458-bdb4-bb1ee3e413fb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s5covid = connection.load_collection(\n",
+    "    \"SENTINEL_5P_L2\",\n",
+    "    temporal_extent=[\"2020-06-01\", \"2021-06-30\"],\n",
+    "    spatial_extent={\"west\": 77.11, \"south\": 28.56, \"east\": 77.29, \"north\": 28.69},\n",
+    "    bands=[\"NO2\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "1b56dda3-0711-4e2c-aec8-82bdf14cbe4e",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Now aggregate by day to avoid having multiple data per day\n",
+    "s5covid = s5covid.aggregate_temporal_period(reducer=\"mean\", period=\"day\")\n",
+    "\n",
+    "# let's create a spatial aggregation to generate mean timeseries data\n",
+    "s5covid = s5covid.aggregate_spatial(reducer=\"mean\", geometries=aoi)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad81db93-619d-443e-bafc-71fc9df25a89",
+   "metadata": {},
+   "source": [
+    "### 2. Let's repeat the same process for Post Covid Situation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "4ec633b6-de35-499b-a07b-6d134c2c64ed",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create a datacube for period after COVID lockdowns\n",
+    "\n",
+    "s5post = connection.load_collection(\n",
+    "    \"SENTINEL_5P_L2\",\n",
+    "    temporal_extent=[\"2022-06-01\", \"2023-06-30\"],\n",
+    "    spatial_extent={\"west\": 77.11, \"south\": 28.56, \"east\": 77.29, \"north\": 28.69},\n",
+    "    bands=[\"NO2\"],\n",
+    ")\n",
+    "\n",
+    "# Now aggregate by day to avoid having multiple data per day\n",
+    "s5post = s5post.aggregate_temporal_period(reducer=\"mean\", period=\"day\")\n",
+    "\n",
+    "# Now create a spatial aggregation to generate mean timeseries data\n",
+    "s5post = s5post.aggregate_spatial(reducer=\"mean\", geometries=aoi)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c670bad3-00a5-44f2-8104-99e78e16098e",
+   "metadata": {},
+   "source": [
+    "Finally we execute them as batch jobs and download the results as CSVs. They are further plotted as shown below to study the data."
```

Comment:
you say CSV here, but use file extension ".nc", suggesting netCDF

Reply:
updated